### ISSUE TITLE
version number creates wrong impression

### ISSUE THREAD

**Issue by ThomasWaldmann** [jborg/attic#212]
Would you trust your data to a version 0.x software? ;)

I think the version number should get a significant increase as it doesn't reflect the quality, capability and that it is in development (and I suspect also in usage) since about 5 years. I also didn't see any major malfunctions.

So, how about something like 0.9 or 1.0.0 beta/rc?

**Comment by anarcat** [comment]
i think that's a great idea. however the reality of it is that Jonas is developing this in his free time, and that just got cut down dramatically (see [this recent post](http://librelist.com/browser//attic/2015/1/27/status-update-i-m-still-alive/) for context). Jonas has also fairly explicitly stated that this is a personal project that he maintains in his free time (see also [this comment](https://github.com/jborg/attic/issues/73#issuecomment-41947730)) and since he's the sole maintainer of the project right now, it makes me somewhat concerned over the viability of the project, even though I use it for my personal backups and we are testing it at work.

**Comment by ThomasWaldmann** [comment]
Yes, increasing the "bus factor" would be a good thing. That said, this is much less an issue for FOSS projects (compared to closed-source or non-free projects) - esp. if you use a DVCS system.

As long as people are interested in the code and capable of working with it, it will live on.

BTW, I did some researches on the web and on twitter. People are really impressed and happy users AFAICS.

**Comment by anarcat** [comment]
see #217 for the bus factor discussion, which should probably be separate. :)

i also have heard generally good things about attic, although I am a bit worried about cache, performance and consistency issues. i have had one report of a friend that has its cache regenerated at each run, which makes backups slower than they should be. there were also reports of backup time increasing linearly with time [here](http://librelist.com/browser//attic/2014/12/27/times-increase-linearly-with-backups-done/) and [here](https://www.debian-administration.org/article/712/An_introduction_to_the_attic_backup_program#comment_3). there's also #102 for performance. i am also worried about various python exceptions that are documented in the issue queue (#163, #205, #196, #161, #139 and more)...

i guess a cleanup of the queue would be in order before such a release, because there seems to be a lot of unconfirmed issues in the above list.
